pkg/lib/queueinformer/resourcequeue.go

```diff
@@ -34,6 +34,18 @@ func (r *ResourceQueueSet) Set(key string, queue workqueue.RateLimitingInterface
 	r.queueSet[key] = queue
 }
 
+func (r *ResourceQueueSet) RequeueEvent(namespace string, resourceEvent kubestate.ResourceEvent) error {
+	r.mutex.RLock()
+	defer r.mutex.RUnlock()
+
+	if queue, ok := r.queueSet[namespace]; ok {
+		queue.AddRateLimited(resourceEvent)
```

Comment:
Really what I wanted to do here is `queue.AddAfter(resourceEvent, 10 * time.Second)`, but I think what is here works okay at least. I figured a timeout wouldn't be allowed, but it would avoid some potential requeueing due to lingering CSVs.